clients/java/src/test/java/com/google/myanmartools/TransliterateTest.java

```diff
@@ -0,0 +1,722 @@
+      "ခ ေဘယ်",
+      "ခ ဘေယျ"
+    },
+    {
+      "ခ⁠ဲဘယ်",
+      "ခဲဘယျ"
+    },
+    {
+      "ခ⁠ဲဘယ်",
+      "ခဲဘယျ"
+    },
+    {
+      "ခ‍ှဘယ်",
+      "ခှဘယျ"
+    },
+    {  // #66
+      "ဃ ၠဃၠဃ ႓ဃ‍႓ဃ⁠႔\u1006",
+      "ဃ္ကဃ္ကဃ္ဘဃ္ဘဃ့\u2060\u1006"
+    },
+    {
+      "\u1001\u0020\u1031\u1018\u101a\u1039",
+
+      "ခ ဘေယ်"
+    },
+    {  // #68
+      "၁၉၄၈ ခုႏွစ္​၊ ဒီဇင္​သာလ ၁၀ ရက္​​ေန႔တြင္​ ကဓၻာ့ကုလသမဂအဖဲြ႔ အတည္​ျပဳ၍ ​ေၾကညာလိုက္​ရာ",
+     "၁၉၄၈ ခုနှစ်​၊ ဒီဇင်​သာလ ၁၀ ရက်​​နေ့တွင်​ ကဓ္ဘာ့ကုလသမဂအဖွဲ့ အတည်​ပြု၍ ​ကြေညာလိုက်​ရာ"
+    },
+    {
+      "၁၉၄၈ ခုႏွစ​္၊ ဒီဇင​္သာလ ၁၀ ရက​္ေန႔တ​ြင္ ကဓၻာ​့ကုလသမဂအဖဲြ႔",
+
+      "၁၉၄၈ ခုနှစ်၊ ဒီဇင်သာလ ၁၀ ရက်နေ့တွင် ကဓ္ဘာ့​ကုလသမဂအဖွဲ့"
+    },
+    {
+      "\u103b\u103b\u1000",
+
+      "\u1000\u103c"
+    },
+    {
+      "\u103b\u103b\u107e\u1000",
+
+      "\u1000\u103c"
+    },
+    {
+      "\u107f\u1084\u1082\u1000",
+
+      "\u1000\u103c"
+    },
+    {
+      "\u103b\u107f\u1080\u1083\u1084\u1081\u1082\u1000",
+
+      "\u1000\u103c"
+    },
+    {
+      "\u1000\u102d\u102d",
+
+      "\u1000\u102d"
+    },
+    {
+      "\u1001\u102e\u102e",
+
+      "\u1001\u102e"
+    },
+    {
+      "\u1002\u102f\u102f",
+
+      "\u1002\u102f"
+    },
+    {
+      "\u1003\u1030\u1030",
+
+      "\u1003\u1030"
+    },
+    {
+      "\u1004\u1032\u1032",
+
+      "\u1004\u1032"
+    },
+    {
+      "\u1005\u1033\u1033",
+
+      "\u1005\u102f"
+    },
+    {
+      "\u1005\u1033\u1033",
+
+      "\u1005\u102f"
+    },
+    { // #81
+      "\u1005\u1030\u1030", //    "\u1005\u1034\u1034",
+
+      "\u1005\u1030"
+    },
+    {
+      "\u1006\u1036\u1036",
+
+      "\u1006\u1036"
+    },
+    {
+      "\u1007\u1037\u1037",
+
+      "\u1007\u1037"
+    },
+    {
+      "\u1009\u1039\u1039",
+
+      "\u1009\u103a"
+    },
+    {
+      "\u100a\u103a\u103a",
+
+      "\u100a\u103b"
+    },
+    {
+      "\u103b\u100b\u103b\u103b",
+
+      "\u100b\u103c"
+    },
+    {
+      "\u100d\u103d\u103d",
+
+      "\u100d\u103e"
+    },
+    {
+      "\u100e\u103e\u103e",
+
+      "\u100e\u103e"
+    },
+    {
+      "\u103b\u107f\u1080\u1081\u100f",
+
+      "\u100f\u103c"
+    },
+    {
+      "ျျမန္​မာကာမန္​မာကာ",
+      "မြန်​မာကာမန်​မာကာ"
+    },
+    {
+      "အခ်စ္႔႔သီေသာ pdf",
+      "အချစ့်သီသော pdf"
+    },
+    {
+      "တကယ္ဆို အျငိႈးေတြမဲ႔ေသာလမ္းေသာလမ္းမွာ တိုႈျပန္ဆံုျကတဲ႔အခါ ",
+
+      "တကယ်ဆို အငြှိုးတွေမဲ့သောလမ်းသောလမ်းမှာ တှိုပြန်ဆုံကြတဲ့အခါ "
+    },
+    {
+      "ျမန္္မာကို",
+      "မြန်မာကို"
+    },
+    {
+      "တာ ့မိမိရဲ့",
+      "တာ့ မိမိရဲ့",
+    },
+    // Odd spacing and strange combinations, for more testing later.
+    /*
+      {
+      "ညီနဲ ့်",
+      "ညီနျဲ့"
+      },
+      {
+      "တာ ့မိမိရဲ့",
+      "တာ့ မိမိရဲ့",
+      },
+      {
+      "ထားေပး်ခင္းအား်ဖင္ ့",
+      "ထားပျေးခင်းအျားဖင့်"
+      },
+      {
+      "လယၱီေ်မ",  // Found in searches
+      "လယ္တျီမေ"
+      }
+    */
+
+  };
+
+
+  // Special cases with digits at the beginning of input.
```

Comment:
Why are these separate from the main tests?  Please add a comment explaining why.

I notice that they are tested Z2U but not U2Z.